### PR TITLE
RNs-4.10.40 Added with Bug Fix Update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.10.40**
Added with Bug Fix Update

**Version(s):**
* PR applies only to enterprise-4.10

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4470

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.10.40/release_notes/ocp-4-10-release-notes.html#ocp-4-10-40

**Additional information:**
`shipped live`

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3572,4 +3572,28 @@ For more information, see xref:../authentication/bound-service-account-tokens.ad
...
+
+[source,terminal]
+----
+$ oc adm release info 4.10.40 --pullspecs
+----
+
+[id="ocp-4-10-40-bug-fixes"]
+==== Bug fixes
+
+* Previously, `noAllowedAddressPairs` settings applied to all subnets on the same network instead of the one matching its corresponding subnet. Due to this, all ports had no address pairs. With this fix, `noAllowedAddressPairs` is now in its proper setting, ensuring each port is filtered to match its corresponding subnet. (link:https://issues.redhat.com/browse/OCPBUGS-1951)[*OCPBUGS-1951*])
```
Comment (tmalove) [inline release_notes/ocp-4-10-release-notes.adoc:3594]: I suggest changing this to 'Before this update...' from 'Previously'. Although I can't find  the source information as a reference, this came up when running Vale for release notes.

Comment (Wiharris) [reply release_notes/ocp-4-10-release-notes.adoc]: Done! Updates have been made. :)

**Pull request merged**